targets/TARGET_Freescale/TARGET_MCUXpresso_MCUS/TARGET_K66F/spi_api.c

```diff
@@ -32,6 +32,24 @@ static SPI_Type *const spi_address[] = SPI_BASE_PTRS;
 /* Array of SPI bus clock frequencies */
 static clock_name_t const spi_clocks[] = SPI_CLOCK_FREQS;
 
+SPIName spi_get_peripheral_name(PinName mosi, PinName miso, PinName sclk) {
```

Comment:
I see line 54 following our code style, this function should as well `{` on the new line

Reply:
also for other files, same one